tests/Alchemy/Tests/Phrasea/Authentication/Provider/FactoryTest.php

```diff
@@ -23,17 +23,16 @@ public function testBuild($name, $options, $expected)
 
         $factory = new Factory($generator, $session);
 
-        $this->assertInstanceOf($expected, $factory->build($name, $options));
+        $this->assertInstanceOf($expected, $factory->build($id, $type, $display, $title, $options));
     }
 
     public function provideNameAndOptions()
     {
         return [
-            ['github', ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Github'],
-            ['google-plus', ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\GooglePlus'],
-            ['linkedin', ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Linkedin'],
-            ['twitter', ['consumer-key' => 'id', 'consumer-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Twitter'],
-            ['viadeo', ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Viadeo'],
+            ['github-test'     , 'Github'    , true, 'Gihtub'  , ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Github'],
```

Comment:
You can use the `Alchemy\Phrasea\Authentication\Provider\Github::class` form.
or `Github::class` with an import.
```suggestion
            ['github-test'     , 'Github'    , true, 'Gihtub'  , ['client-id' => 'id', 'client-secret' => 'secret'], 'Alchemy\Phrasea\Authentication\Provider\Github'],
            ['github-test'     , 'Github'    , true, 'Gihtub'  , ['client-id' => 'id', 'client-secret' => 'secret'], Github::class],
```